interface/forms/vitals/C_FormVitals.class.php

```diff
@@ -33,18 +54,32 @@ public function __construct($template_mod = "general")
         $this->assign("STYLE", $GLOBALS['style']);
 
       // Options for units of measurement and things to omit.
-        $this->assign("units_of_measurement", $GLOBALS['units_of_measurement']);
-        $this->assign("gbl_vitals_options", $GLOBALS['gbl_vitals_options']);
+        $this->assign("units_of_measurement", $this->units_of_measurement);
+
+        $this->assign("MEASUREMENT_METRIC_ONLY", FormVitals::MEASUREMENT_METRIC_ONLY);
+        $this->assign("MEASUREMENT_USA_ONLY", FormVitals::MEASUREMENT_USA_ONLY);
+        $this->assign("MEASUREMENT_PERSIST_IN_METRIC", FormVitals::MEASUREMENT_PERSIST_IN_METRIC);
+        $this->assign("MEASUREMENT_PERSIST_IN_USA", FormVitals::MEASUREMENT_PERSIST_IN_USA);
+
+//        $this->assign("gbl_vitals_options", $GLOBALS['gbl_vitals_options']);
+        $this->assign("hide_circumferences", $GLOBALS['gbl_vitals_options'] > 0);
 
         // Assign the CSRF_TOKEN_FORM
         $this->assign("CSRF_TOKEN_FORM", CsrfUtils::collectCsrfToken());
     }
 
+    /**
+     * TODO: can this function be removed?
+     * @deprecated
+     * @return mixed
+     */
     public function default_action_old()
```

Comment:
if not being used then would remove it :)

![tenor (3)](https://user-images.githubusercontent.com/278968/126412614-957ba469-962c-4b5c-a960-3e4a5d13066e.gif)